docs/sdk/SDK_API_DEFINITION.md

```diff
@@ -0,0 +1,205 @@
+# Mesh Data Plane SDK Definition
+
+## mesh_create_client()
+```c
+int mesh_create_client(MeshClient **mc,
+                       MeshClientConfig *cfg)
+```
+Creates a new mesh client from the given configuration structure.
+
+#### Parameters
+* `[OUT]` `mc` – Address of a pointer to a mesh client structure.
+* `[IN]` `cfg` – Pointer to a mesh client configuration structure.
```

Comment:
I think out/in "prefix" can be shown also in the function declaration itself in the code.

Reply:
Agree. This is what I'm working on.

Reply:
Fixed in eae8b083e0210567994eb379b5e228742451c44b.